DirectEmployers/MyJobs
# [PD-1952]  restore report naming

## Changes

There are three changes in this PR:

1. The addition of managing report names in the report setup.
2. Merging in some of the newer standard components.
3. Cleaning up ReportFinder and ReportConfiguration.

## Test

Go to https://secure.my.jobs/reports/view/dynamicoverview/

1. Create and download a report.
2. Try creating a report with a different name. Name should show up in the report list.
3. Try creating a report with a blank name. Should get an error with standard styling.
